### PR TITLE
TC06 - Additional pages display once user logged in

### PR DESCRIPTION
**Pull request by sattdn21-01-02**

### CODE REVIEW

File: src/main/java/page_objects/ChangePasswordPage.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package page_objects;
+
+import helper.element_helper.Label;
+import org.openqa.selenium.By;
+
+public class ChangePasswordPage extends GeneralPage {
+
+    //Elements
+    private final Label lblChangePassword = new Label(By.cssSelector("[align='center']"));
```
Comment (fey031) [inline src/main/java/page_objects/ChangePasswordPage.java:9]: for title in this page, we should use //div[@id='content']//h1.
If there are more than one locator that have attribute `[align='center']`, your locator will not be unique

File: src/main/java/page_objects/MyTicketPage.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package page_objects;
+
+import helper.element_helper.Label;
+import org.openqa.selenium.By;
+
+public class MyTicketPage extends GeneralPage {
+
+    //Elements
+    private final Label lblMyTicket = new Label(By.cssSelector("[align='center']"));
```
Comment (fey031) [inline src/main/java/page_objects/MyTicketPage.java:9]: same above

File: src/test/java/com/logigear/LoginTest.java [hunk 3]
```diff
@@ -2,20 +2,46 @@
...
+        LoggerHelper.startTestCase("TC03 - Additional pages display once user logged in");
+
+        homePage.goToLoginPage();
+        Account account = new Account(Constant.USERNAME, Constant.PASSWORD);
+        loginPage.login(account);
+
+        homePage.goToMyTicketPage();
+        String actualMsgMyTicketPage = myTicketPage.getMyTicketTitle();
+        String expectedMsgMyTicketPage = Constant.MY_TICKET_TITLE;
+        Assert.assertEquals(actualMsgMyTicketPage, expectedMsgMyTicketPage);
```
Comment (fey031) [inline src/test/java/com/logigear/LoginTest.java:34]: missing assert message

File: src/test/java/com/logigear/LoginTest.java [hunk 4]
```diff
@@ -2,20 +2,46 @@
...
+    public void TC06() {
+        LoggerHelper.startTestCase("TC03 - Additional pages display once user logged in");
+
+        homePage.goToLoginPage();
+        Account account = new Account(Constant.USERNAME, Constant.PASSWORD);
+        loginPage.login(account);
+
+        homePage.goToMyTicketPage();
+        String actualMsgMyTicketPage = myTicketPage.getMyTicketTitle();
+        String expectedMsgMyTicketPage = Constant.MY_TICKET_TITLE;
```
Comment (fey031) [inline src/test/java/com/logigear/LoginTest.java:33]: if you define this message in Constant, don't need declare new variable for it

File: src/test/java/com/logigear/LoginTest.java [hunk 5]
```diff
@@ -2,20 +2,46 @@
...
+        Assert.assertEquals(actualMsgMyTicketPage, expectedMsgMyTicketPage);
+
+        homePage.goToChangePasswordPage();
+        String actualMsgChangePasswordPage = changePasswordPage.getChangePasswordTitle();
+        String expectedMsgChangePasswordPage = Constant.CHANGE_PASSWORD_TITLE;
```
Comment (fey031) [inline src/test/java/com/logigear/LoginTest.java:38]: if you define this message in Constant, don't need declare new variable for it

File: src/test/java/com/logigear/LoginTest.java [hunk 6]
```diff
@@ -2,20 +2,46 @@
...
+        Assert.assertEquals(actualMsgChangePasswordPage, expectedMsgChangePasswordPage);
```
Comment (fey031) [inline src/test/java/com/logigear/LoginTest.java:39]: missing assert message

**Review by fey031 — CHANGES_REQUESTED** [review]

File: src/main/java/page_objects/ChangePasswordPage.java (hunk 1 above)
Comment (sattdn21-01-02) [reply src/main/java/page_objects/ChangePasswordPage.java:9]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/LoginTest.java (hunk 4 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/LoginTest.java:33]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/LoginTest.java (hunk 6 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/LoginTest.java:39]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/LoginTest.java (hunk 3 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/LoginTest.java:34]: Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.

File: src/test/java/com/logigear/LoginTest.java [hunk 7]
```diff
@@ -71,4 +74,24 @@ public void TC05() {
         Assert.assertEquals(actualMsg, Constant.MAXIMUM_LOGIN_ATTEMPTS_WARNING_MSG, "Error message fails to display!");
     }
 
+    @Test(description = "TC06 - Additional pages display once user logged in")
+    public void TC06() {
+        LoggerHelper.startTestCase("TC03 - Additional pages display once user logged in");
```
Comment (LoliSlayerMC) [inline src/test/java/com/logigear/LoginTest.java:79]: TC06

**Review by LoliSlayerMC — CHANGES_REQUESTED** [review]

File: src/test/java/com/logigear/LoginTest.java (hunk 7 above)
Comment (sattdn21-01-02) [reply src/test/java/com/logigear/LoginTest.java:79]: So sorry, that my fault. I will check it now

**Review by LoliSlayerMC — APPROVED** [review]
LGTM

**Pull request merged**